packages/zend-exception/library/Zend/Exception.php

```diff
@@ -42,7 +42,7 @@ class Zend_Exception extends Exception
      */
     public function __construct($msg = '', $code = 0, Exception $previous = null)
     {
-        if (version_compare(PHP_VERSION, '5.3.0', '<')) {
+        if (PHP_VERSION_ID < 50300) {
```

Comment:
There are some code that checks for php 5.3:

```php
        if (PHP_VERSION_ID < 50300) {
            parent::__construct($msg, (int) $code);
            $this->_previous = $previous;
        } else {
            parent::__construct($msg, (int) $code, $previous);
        }
```

but as we require only 5.3, perhaps drop such low version checks?

Reply:
go ahead and clean it up if you care (and have time for it) please :)

Reply:
Will have to go manually over each check anyway, as could not find automation.

Reply:
the code part is done and ready for merge:
- https://github.com/zf1s/zf1/pull/54

tests part needs more effort, as there are a lot of checks, so created as separate PR:
- https://github.com/zf1s/zf1/pull/55